figs/stanplots/analyses_wanang.R

```diff
@@ -34,9 +46,121 @@ l1W_ranef <- brm(log(BIO) ~ TREAT + (1+TREAT|GARDEN),
            control = list(adapt_delta = 0.99),
            file="bioAllrenef")
 
-# Summaries
-summary(l1W)
-summary(l1W_ranef)
+
+# random intercepts and slopes --------------------------------------------
+
+#define model 
+#does changing contrasts change the model?
+levels(AllTestData$TREAT)
+#contrasts(AllTestData$TREAT) <- contr.helmert(n = 6)
+
+bio_lnorm_bf <- bf(BIO ~ 1 + TREAT + (1 + TREAT|GARDEN), family = lognormal())
+brms::get_prior(bio_lnorm_bf, data = AllTestData)
+## Set some priors 
+priors <- c(set_prior("normal(4, 2)", class = "Intercept"),
+            set_prior("normal(0, 3)", class = "b"),
+            set_prior("normal(0, 4)", class = "sd"),
+            set_prior("normal(0, 3)", class = "sigma"),
+            set_prior("lkj(2)", class = "cor"))
```

Comment:
These priors are a lot tighter than the ones that you set, but a) they can be played with b) they're still _very_ broad.